msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md

```diff
@@ -68,7 +68,7 @@ import path from "path";
 
 Add AI capabilities to your existing app or a new Bot Framework app.
 
-**Planner**: OpenAI planner is the main component that calls the large language model (LLM) OpenAI or Azure OpenAI. The OpenAI API is powered by a diverse set of models with different capabilities. You can also make limited customizations to our original base models for your specific use case.
+**Planner**: A planner is a function that takes in a user's ask and returns a plan on how to accomplish the request. Planner is the main component that calls the large language model (LLM) OpenAI or Azure OpenAI. The OpenAI API is powered by a diverse set of models with different capabilities. You can also make limited customizations to our original base models for your specific use case.
```

Comment:
**Planner**: A planner is a feature that accepts a user's request and generates a strategy to fulfill it. The planner is the primary component that invokes the Large Language Model (LLM) from OpenAI or Azure OpenAI. The OpenAI API is driven by an array of models, each with unique capabilities. For your specific use case, you might make minor customizations to our original base models.

Reply:
Updated.